### ISSUE TITLE
Default sort

### ISSUE THREAD

**Issue by TheDarkITA** [lacodix/laravel-model-filter#56]
What is the best way to set default sort if nothing is passed in query string?

**Comment by renky** [comment]
To be honest: I already came to the same problem...
In my case I indeed use it currently in a Livewire-installations where i can set a default-querystring, so this solution solved the problem for me.

But you are right: without such a default-query, there is no solution currently to influence the sorting...
Indeed currently it is NOT applying the sort-Query when nothing is given (even if an empty array is given).

I'm open to suggestions: one solution could be to extend the sortable array like that:
```
$sortable = [
    'created_at' => 'desc',
    'title',
    ...
];
```
if a direction is given, then this is the default sorting, if none is given, it is 'asc' (thats already like now: if you have 3 columns, but only insert one for sorting, the other both are sorted in 'asc'-direction)

additionally I'd have to change the behaviour of the scops. as already said, currently it is NOT applied, if nothing is given. With that solution, we'd have to accept that the sorting query is always applied...

maybe we would also have to think about a setting if a default sorting is wished or not... we could say: if a direction in sortable is given, then the user always wants sorting, and if none is given, it should behave like today...

I'm not sure if this all makes sense... please feel free to comment on it.

**Comment by renky** [comment]
@TheDarkITA Hey, i just released a new version v1.11.x that contains the default-sorting-option

https://www.lacodix.de/docs/laravel-model-filter/v1/basic-usage/sort#content-default-sorting
